packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -13,8 +13,10 @@ export interface MenuButtonState {
 }
 
 export interface MenuButtonItemProps extends ContextualMenuItemProps {
-  submenu?: boolean;
+  hasSubmenu?: boolean;
   submenuItems?: ContextualMenuItemProps[];
+  submenu?: SubmenuProps;
```

Comment:
naming nit: It might be clearer to have "Props" in the prop name here, so "submenuProps," to make the purpose of this prop clearer

Reply:
fixed

Reply:
I don't see this fixed, not sure if the change didn't make it through or if I'm not seeing the change...

Reply:
WHoops was looking at the wrong commit, nvm